app/lib/validation_utils.rb

```diff
@@ -0,0 +1,5 @@
+module ValidationUtils
+  def valid_year?(year)
+    year.to_s.match?(/^[1-9]\d{3}$/)
```

Comment:
This module does not have unit tests for itself but it's tested as part of the model specs; based on our conversation earlier in the tech debt discussion this feels fine as we shouldn't be adding redundant tests.

Reply:
maybe it might be worth add couple of test fot the reg exp

Reply:
Regexps are quite "susceptible", so at the end, I added some small tests for it,
I did it also to set up a spec, so next time will be easier to add examples on it.